src/components/composites/TopHeader/index.js

```diff
@@ -0,0 +1,52 @@
+import { ArrowBack } from '@mui/icons-material';
+import { AppBar, Box, Button, Toolbar, Typography } from '@mui/material';
+import React from 'react';
+import HideOnScroll from '../../utility/HideOnScroll';
+import './topHeader.css'
+
+export default function TopHeader(props) {
```

Comment:
There shouldn't be different Header files - the other one should be removed, or they need to be connected somehow. Maybe this should be "MobileHeader"?

Reply:
I think a **`Modal`** might be a better fit for the **_Splash Screen_**

Reply:
MobileHeader implemented